src/main/java/com/google/jspecify/nullness/NullSpecAnnotatedTypeFactory.java

```diff
@@ -1983,7 +1975,6 @@ private enum Present {
                 "org.jetbrains.annotations.Nullable",
                 "org.jmlspecs.annotation.Nullable",
                 "org.jspecify.annotations.Nullable",
-                "org.jspecify.nullness.Nullable",
```

Comment:
What's up with this entire list? If this is the reference checker for JSpecify annotations, should it understand anything else at all?